cocos/asset/assets/render-texture.ts

```diff
@@ -168,6 +172,9 @@ export class RenderTexture extends TextureBase {
         _windowInfo.width = this._width;
         _windowInfo.height = this._height;
         _windowInfo.renderPassInfo = info && info.passInfo ? info.passInfo : passInfo;
+        _windowInfo.externalResLow = info && info.externalResLow ? info.externalResLow : 0;
+        _windowInfo.externalResHigh = info && info.externalResHigh ? info.externalResHigh : 0;
+        _windowInfo.externalFlag = info && info.externalFlag ? info.externalFlag : TextureExternalFlag.NONE;
```

Comment:
Please add some comments to explain the usage of externalResXXX and externalFlag.
Developers might not understand the reason behind the scene.